src/platform/routes/LandingPage/index.module.scss

```diff
@@ -0,0 +1,46 @@
+@import "~assets/styles/global";
+
+.landingPageContainer {
+  padding: 20px 30px;
```

Comment:
You need to increase the padding from bottom, for all screens ⚠️

Reply:
the code here is not the problem.
there is a component for the buttons in the Home page I gave it the bottom padding but not to the home page as it didn't solve the solution